src/main/java/de/bonndan/nivio/api/ApiController.java

```diff
@@ -133,8 +133,7 @@ public ProcessLog indexLandscape(
             @RequestHeader(name = "format") String format,
             @RequestBody String body
     ) {
-        LandscapeDescription dto = new LandscapeDescription();
-        dto.setIdentifier(identifier);
+        LandscapeDescription dto = new LandscapeDescription(identifier, "", null);
```

Comment:
Is this meaningful?

This is for `@RequestMapping(path = "/landscape/{identifier}/services", method = RequestMethod.POST)`, which I think triggers the indexing of a landscape provided by the POST body? Is this correct? Or are just Items added?

Reply:
This is for adding items via POST body. It should also trigger reindexing. While you're at it, you might consider renaming the path to "/items", because "/services" is not used as term anymore and narrows the usage too much (although this is not scope of the ticket).

Reply:
That is fine, I renamed the endpoint.

So it is totally ok to have a LandscapeDescription with an identifier, but without a human readble name - correct?

Reply:
Yes.